src/App.js

```diff
@@ -23,18 +24,26 @@ function App() {
     setToken(localStorage.getItem('userToken'));
   }, [push]);
 
+  const background = {
+    backgroundImage: `url(${lemons})`,
+  };
+
   return (
     <ListContextProvider>
       <div
+        style={background}
         className="
-          border-b-8 border-teal-600 
-          flex-row max-w-4xl mx-auto 
+          border-b-8 border-teal-600
+          bg-center
+          bg-cover 
+          flex-row mx-auto 
           bg-gray-100
-          shadow-2xl 
           relative h-screen"
       >
-        <header className="p-10">
-          <h1 className="text-5xl text-center text-teal-800">Smart Shopping</h1>
+        <header className="p-4">
+          <h1 className="text-3xl text-center text-teal-800 sm:text-4xl">
```

Comment:
Lily's & Shajia's PR had this too, a class name with a `:` in it. I've never seen that before. Is that valid?

Reply:
Yes, `sm:` creates responsive utilities for smaller breakpoints. It's a tailwind feature.

Reply:
It is! It's how Tailwind handles things like breakpoints. https://tailwindcss.com/docs/responsive-design